### ISSUE TITLE
question about learingRate in cnn_cifar_init.m 

### ISSUE THREAD

**Issue by dafeifei2016** [vlfeat/matconvnet#807]
What's the meaning of "learingRate" in struct "conv" of cnn_cifar_init.m. when i delete the hyperparameter,the net do not work?

**Comment by albanie** [comment]
The learning rate is a parameter that is used as part of the optimisation algorithm for training the networks.  

For details, there is a good introduction to the optimisation algorithms used to train neural nets in [this chapter](http://www.deeplearningbook.org/contents/optimization.html) of the deep learning book.
